src/main/webapp/WEB-INF/templates/user_dashboard.jsp

```diff
@@ -31,7 +40,11 @@
       <div class="row">
         <label for="input-languages">Choose your primary coding language:</label>
         <input class="input-languages" id="input-languages" type="text" name="language"
-               placeholder="Input your primary coding language">
+        <c:choose>
```

Comment:
I don't think you need to make this conditional. It's certainly okay to keep the placeholder regardless in case the user blanks out the field.